charts/openshift-metering/templates/hadoop/_hadoop_config_helpers.tpl

```diff
@@ -108,5 +108,40 @@
       <name>fs.gs.reported.permissions</name>
       <value>733</value>
   </property>
+  <property>
+      <name>fs.s3a.proxy.host</name>
+{{- if and .Values.networking.useGlobalProxyNetworking .Values.presto.spec.config.tls.enabled }}
+      <value>{{ .Values.networking.proxy.config.https_proxy.hostname }}</value>
+  </property>
+  <property>
+      <name>fs.s3a.proxy.port</name>
+      <value>{{ .Values.networking.proxy.config.https_proxy.port }}</value>
+  </property>
+  <property>
+      <name>fs.s3a.proxy.username</name>
+      <value>{{ .Values.networking.proxy.config.https_proxy.username }}</value>
+  </property>
+  <property>
+      <name>fs.s3a.proxy.password</name>
+      <value>{{ .Values.networking.proxy.config.https_proxy.password }}</value>
+  </property>
+  <property>
+{{- else if .Values.networking.useGlobalProxyNetworking }}
+      <name>fs.s3a.proxy.host</name>
```

Comment:
You need to swap lines 128 with 129. The `{{- else if ... }}` conditional is splitting up this property configuration.
```suggestion
{{- else if .Values.networking.useGlobalProxyNetworking }}
  <property>
      <name>fs.s3a.proxy.host</name>
```